src/HotChocolate/Core/test/Types.Tests/Types/Relay/__snapshots__/IdAttributeTests.Id_On_Objects_InvalidId.snap

```diff
@@ -1,4 +1,4 @@
 ﻿{
-  "result": "{\n  \"errors\": [\n    {\n      \"message\": \"The ID \\u0060abc\\u0060 has an invalid format.\"\n    }\n  ],\n  \"data\": {\n    \"foo\": null\n  }\n}",
```

Comment:
I don't know why this one changed, nor the one below... doesn't seem like something my changes would've affected.

Reply:
you changed the test schema which now triggers the non-null propagation. So this change seems ok